applications/bountyblok.md

```diff
@@ -0,0 +1,184 @@
+
+* Jan Bui:
+
+  With over 2 decades' experience in technical leadership and a sound vision of the delicate balance between technology and business, Jan has positioned inlineFX at the forefront of the technology landscape. He has worked previously in Montreal, Zurich and London, working and managing high-level teams of front-office software developers at various banks and brokerages such as Credit Suisse, Morgan Stanley and BGC Partners.
+
+* Dimitri Nikolaros:
+
+  As Co-Founder and lead programmer, Dimitri focuses on delivering quality and functional software. He has developed websites that have processed millions of transactions and currently used by thousands of avid users. In addition, Dimitri focuses on client-relationship management and other strategic initiatives.
+
+### Team Member Repos
+
+* https://github.com/bountyblok 
+* https://github.com/diminiko
+
+### Team LinkedIn Profiles (if available)
+
+* https://www.linkedin.com/in/dnikolaros/ Dimitri Nikolaros
+* https://www.linkedin.com/in/jan-bui-4b62383/ Jan Bui
+
+## Development Status
+
+The distribution tool went live on EOS in Q4 2021: https://eos.drop.bountyblok.io. However, this version went live with a limited set of features as we required additional infrastructure at the time.
+
+## Development Roadmap
+
+### Overview
+
+We are looking to expand our features and offer what we have right now on the WAX version and some of these features include:
+
+* Ability to query accounts who completed entire sets
+* Ability to query accounts via schema attributes 
+* Staking - a lot of projects stake their NFTs to earn rewards, for example "wombatmaster"
+* Floor Sweeping (ability to buy a large number of the cheapest listed assets)
+* Accounts by Instagram comments
+* and many more
+
+The costs below include on-going maintenance, and customer support for the next 12 months. We offer support via telegram and email 7 days a week.
+
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  4
+- **Total Costs:** 100,000 USD
+
+### Milestone 1 SETUP INFRA + BACKEND
+
+- **Estimated duration:** 1 month
+- **FTE:**  2
+- **Costs:** 30,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 
+| 0b. | Documentation | Not applicable here |
+| 0c. | Testing Guide | Not applicable here |
+| 0d. | Docker | Not applicable here |
+| 0e. | Article | Not applicable here |
+| 1. | EOS SHIP Setup | We will setup infrastructure and run our own EOS SHIP nodes |  
+| 2. | Atomicassets DB | We will setup infrastructure in order to run our own Atomicassets DB and API | 
+| 3. | New Backend | Deploy a new backend instance |   
+
+### Milestone 2 UI Components
+
+- **Estimated duration:** 1 month
+- **FTE:**  4
+- **Costs:** 70,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 
+| 0b. | Documentation | Not applicable here |
```

Comment:
The license is listed as Apache 2.0 and ENF grants carry a requirement that code developed as a result of a grant must be open sourced.  I don't see the existing WAX UI/UX code in a repo (which is fine) is the intent to port *and* opensource that UI/UX as part of this grant?  

It is unclear if the SQL Module is a port or new work, either way those modifications would carry an expectation of open source. 

If so, I would strongly encourage documentation as part of the deliverable and, depending on your development environment, either some sort of containerized deployment or guide for integrating with the preferred hosting environment (eg, Vercel).

Reply:
Hi Dimitri,
Unfortunately the ENF Grant Program is only for **open-source** projects.  If you cannot open-source it, we will unfortunately need to reject this grant.  We hope you will reconsider changing it to open-source.
[See requirements](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/minimum-requirements.md).